apepay/__init__.py

```diff
@@ -51,16 +52,21 @@ def validate(self, creator, token, amount_per_second, reason) -> bool:
 
 class StreamManager(BaseInterfaceModel):
     address: AddressType
+    contract_type: Optional[ContractType] = None
```

Comment:
I want to somehow bundle the manifest/contract type with the package so that we always have this as a backup